build.gradle

```diff
@@ -37,6 +37,7 @@ allprojects {
     apply plugin: 'nebula.dependency-recommender'
 
     dependencyRecommendations {
+        strategy OverrideTransitives
```

Comment:
we should just remove the usage of this plugin considering this library has zero dependencies (and we'll probably want to keep it that way).